src/ontology/cl-edit.owl

```diff
@@ -31105,6 +31106,16 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37292694") Annotation(ob
 AnnotationAssertion(rdfs:label obo:CL_4030065 "L6 intratelencephalic projecting glutamatergic neuron")
 EquivalentClasses(obo:CL_4030065 ObjectIntersectionOf(obo:CL_4023040 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0005395)))
 
+# Class: obo:CL_4030066 (ureteric bud cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:17133361") Annotation(oboInOwl:hasDbXref "PMID:24183650") Annotation(oboInOwl:hasDbXref "PMID:9374839") obo:IAO_0000115 obo:CL_4030066 "An epithelial cell that is part of a ureteric bud. A ureteric bud cell has the potential to induce metanephric mesenchymal cells to proliferate and convert to epithelia that form renal tubules via: (1) the secretion of multiple diffusible growth factors that rescue renal progenitors from apoptosis and (2) contact-dependent mechanisms that induce mesenchymal-epithelial conversion.")
```

Comment:
> A ureteric bud cell has the potential to induce metanephric mesenchymal cells to proliferate and convert to epithelia that form renal tubules via

These are two separate processes, and from the reference, during proliferation there is not epithelialization:
![image](https://github.com/obophenotype/cell-ontology/assets/94959119/53c0fd9b-4a98-4690-969e-383424640b54)
Consider clarifying that both processes are independent.

> the secretion of multiple diffusible growth factors that rescue renal progenitors from apoptosis

As it is written, I understand that this is the mechanism of how UB cells induce proliferation. However, while growth factors can induce this proliferation, there is only reference to the apoptosis rescue. Consider either removing the "via" fro proliferation, or adding the mechanism of proliferation.

Reply:
I think that is a good solution for the second point, thank you. However, in the previous sentence, consider making clear that proliferation/apoptosis inhibition and epithelialization are not occurring simultaneously.

Reply:
> consider making clear that proliferation/apoptosis inhibition and epithelialization are not occurring simultaneously.

Is that not clear by breaking it into 2 parts in the definition? Two separate mechanisms: (1) proliferation via signalling from diffusible growth factors and (2) epithelialisation via contact-dependent mechanisms.

Reply:
> How about the following:
"... (1) the secretion of multiple diffusible growth factors that rescue renal progenitors from apoptosis and stimulate them to proliferate and (2) ..."

Once these changes are implemented, I will approve the PR.